src/features/PostState/postsReducer.js

```diff
@@ -3,17 +3,27 @@ const postsReducer = (state, action) => {
         return {
             ...state,
             isLoading: true,
+            posts: [],
         };
     }
-    if (action.type === 'FETCH_POSTS_SUCCESSFUL') {
+    if (action.type === 'ADD_PAGE_OF_POSTS') {
```

Comment:
Now that posts are only returned in pages the `ADD_PAGE_OF_POSTS` action is the only action needed.